Aptfile

```diff
@@ -0,0 +1,4 @@
+libsm6
+libxrender1
+libfontconfig1
+libice6
```

Comment:
Why this file is pushed?

Reply:
These are the configurations required to deploy the app to Heroku.